1-index.html

```diff
@@ -34,7 +30,7 @@ <h2>What is Jamulus?</h2>
 <div class="fx-row fx-row-center-xs">
   <div class="fx-col-100-xs fx-col-50-l">
      <h2>Jamulus worldwide</h2>
-    All over the world Jamulus allows huge choirs to sing, loud rock bands to play, and brings folk and classical musicians together. It's being used for remote music lessons,
+    All over the world Jamulus allows huge choirs to rehearse, loud rock bands to play, and brings folk and classical musicians together. It's being used for remote music lessons,
```

Comment:
This rather kills the slightly poetic nature of the original :-) What's wrong with choirs singing? And do rock bands no rehearse? That's not the point, I know, but...